raiden/tests/integration/network/transport/test_matrix_transport.py

```diff
@@ -1000,11 +1022,12 @@ def test_matrix_user_roaming(matrix_transports):
     assert ping_pong_message_success(transport0, transport1)
 
 
+@pytest.mark.xfail(reason="XFail until raiden-network/raiden#4030 is fixed")
```

Comment:
Not sure I get why the tests fail. Did you try extending the timeouts? Messages should arrive at some point, even though only send once via send_raw

Reply:
See #4030 for the details.